src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx

```diff
@@ -5,47 +5,45 @@ metaDescription: 'Manage New Relic and Azure integrations with agents installed
 
 import infrastructureMonitorResources from 'images/monitored-resources.png'
 
-If your team deploys software and cloud services through Azure, our [Azure Native New Relic Service](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg) lets you deploy and manage New Relic directly through the Azure Marketplace. With it, you can: 
+If your team deploys software and cloud services through Azure, our [Azure Native New Relic Service](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg) lets you manage your Azure-New Relic configuration directly from your Azure account. 
```

Comment:
...manage the Azure-New Relic integration, manage and install infrastructure monitoring agents as well as .NET and Java agents directly from the Azure Portal.